_drafts/oryx/orchestration/resources.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Application Resources
+description: Resources of the Oryx Application
+template: concept-topic-template
+---
+
+# Application resources
```

Comment:
Can we drop this file alltogether? It think most of it is covered by the plugin docs, and other stuff should be covered in the presets docs.

Reply:
I thought it makes sense have a dedicated page for it but I don't mind dropping it as well =)

Reply:
we have it covered in presets docs